openapi-specs/schema.json

```diff
@@ -1316,6 +1316,24 @@
           "minimum": 1,
           "maximum": 80,
           "x-nullable": false
+        },
+        "rolesOptions" : {
+          "description": "How roles should be restored",
+          "type": "string",
+          "enum": [
+            "noRestore",
+            "all"
+          ],
+          "default": "all"
+        },
+        "usersOptions" : {
+          "description": "How users should be restored",
+          "type": "string",
+          "enum": [
+            "noRestore",
+            "all"
+          ],
+          "default": "all"
```

Comment:
should we change the default to `noRestore`?

Reply:
done